internal/worker/api/openapi.yml

```diff
@@ -295,8 +295,11 @@ components:
     UpdateJobRequest:
       type: object
       required:
+        - arch
         - result
       properties:
+        arch:
+          type: string
```

Comment:
I think we should avoid changing the worker api if at all possible, this would be a breaking change. Can't we parse the arch from the jobtype? We actually already have globs in the prometheus queries I think to account for the `:$arch` at the end of osbuild jobs.

Reply:
Yeah didn't seem like a great option. The only thing is that only the `osbuild` jobtype has the arch in it, the others don't.
We could either pass in an empty string for the other types (also not great). 

Or maybe we can store the `arch` somewhere else, any suggestions on where though?

Reply:
makes sense to me to only track the arch for arch specific jobs (build)

Reply:
Doh so I didn't realise that prometheus would hide a field if you pass it an empty string.
Updating the PR, it's actually a very simple fix in the end.